### ISSUE TITLE
can not sync blocks after #8949641

### ISSUE THREAD

**Issue by fntz** [paritytech/polkadot#3791]
Hello,
I run for syncing last polkadot (v0.9.9-1):

```
 ./polkadot --base-path /mnt/vlm/kusama --pruning archive --name sync-kusama --rpc-external --rpc-cors all --ws-external --no-telemetry --chain kusama
 ``` 

but I see (around 3+ days) only logs like this:

```
2021-09-05 21:28:47 ⚙️  Syncing, target=#9098065 (9 peers), best: #8949641 (0x135a…dd8f), finalized #8949248 (0x102e…0032), ⬇ 8.6MiB/s ⬆ 541.3kiB/s    
2021-09-05 21:28:52 💤 Idle (15 peers), best: #8949641 (0x135a…dd8f), finalized #8949248 (0x102e…0032), ⬇ 1.8MiB/s ⬆ 923.6kiB/s    
2021-09-05 21:28:54 🔍 Discovered new external address for our node: /ip4/10.2.112.232/tcp/30333/ws/p2p/12D3KooWNsXoRV92W4Lc2Gr4iaew8LeMSVpLKejzBB32wnFovpr1    
2021-09-05 21:28:57 🔍 Discovered new external address for our node: /ip4/10.110.16.19/tcp/30333/ws/p2p/12D3KooWNsXoRV92W4Lc2Gr4iaew8LeMSVpLKejzBB32wnFovpr1    
2021-09-05 21:28:57 ⚙️  Syncing  0.0 bps, target=#9098067 (18 peers), best: #8949641 (0x135a…dd8f), finalized #8949248 (0x102e…0032), ⬇ 318.6kiB/s ⬆ 1.4MiB/s    
2021-09-05 21:28:57 🔍 Discovered new external address for our node: /ip4/10.0.254.61/tcp/30333/ws/p2p/12D3KooWNsXoRV92W4Lc2Gr4iaew8LeMSVpLKejzBB32wnFovpr1    
2021-09-05 21:28:59 🔍 Discovered new external address for our node: /ip4/100.116.217.64/tcp/30333/ws/p2p/12D3KooWNsXoRV92W4Lc2Gr4iaew8LeMSVpLKejzBB32wnFovpr1    

```

I mean no progress after `8949641` block.

OS info: 
`Linux 5.13.12-200.fc34.x86_64` 
volume:
```
/dev/sda            319G  280G       26G  92% /mnt/volume_fra1_03
```
memory:
```
MemTotal:        2013652 kB
```

Could you help? How to skip/handle this block?

**Comment by bkchr** [comment]
Duplicate of: https://github.com/paritytech/polkadot/issues/3778
